package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "types": "index.d.ts",
   "scripts": {
+    "testonly": "tap -J test/*.test.js && npm run test:typescript",
```

Comment:
why this? Can you remove?

Reply:
I found this useful because when I was developing I frequently wanted to see if I'd broken tests. When running that I didn't want something like indentation linting to stop the tests running.

Happy to remove, but that was my reasoning.

Reply:
ok